extensions/pomodoro/src/stats-pomodoro-timer.tsx

```diff
@@ -0,0 +1,61 @@
+import { Detail } from "@raycast/api";
+import { getIntervalHistory } from "../lib/intervals";
+import { usePromise } from "@raycast/utils";
+import { Interval } from "../lib/types";
+
+function countNumberOfCompletedCycle(intervals: Interval[]) {
+  let count = 0;
+  for (const interval of intervals) {
+    for (const part of interval.parts) {
+      if (part.endAt) {
+        count++;
+      }
+    }
+  }
+  return count;
+}
+
+function countTotalFocusTime(intervals: Interval[]) {
+  let total = 0;
+  for (const interval of intervals) {
+    for (const part of interval.parts) {
+      if (part.endAt) {
+        total += interval.length;
+      }
+    }
+  }
+  return total;
+}
```

Comment:
Should this be filtered to only count `interval.type === "focus"`?

Reply:
Can you help here @cchalop1

Reply:
Yes you are right. I'm going to fix it.